.travis.yml

```diff
@@ -1,19 +0,0 @@
-# .travis.yml
-language: python
-
-env:
-  - LUA="lua=5.1.5"
```

Comment:
i don't see this environment variable being set in the GitHub Action file.

Reply:
It is working fine without it @pandafy.
I didn't include lua GH-actions because it will increase build time.

Success Build URL: https://github.com/codesankalp/openwisp-config/runs/2166367774